src/components/Board.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import { determineWin } from '../utils/determineWin'
+import { Moves } from './Moves'
+import { Square } from './Square'
+import { Status } from './Status'
+
+export function Board(): JSX.Element {
+  const [turn, setTurn] = useState(0)
+  const [boardState, setBoardState] = useState([
+    {
+      board: ['', '', '', '', '', '', '', '', ''],
+    },
+  ])
+  const [playerOne, setPlayerOne] = useState(true)
+
+  /*
+    access boardState to record player's moves and save them into array;
+    this accesses the board property within boardState for each turn
+  */
+  const currentBoardState = boardState.slice(0, turn + 1)
+  const currentMove = currentBoardState[currentBoardState.length - 1]
+  const board = currentMove.board.slice(0)
+  const winner = determineWin(board)
+
+  function handleTurn(index: number) {
+    if (determineWin(board)) {
+      return board[index]
```

Comment:
This if is just to prevent game logic if we are already in a win state. You don't need to return anything here
```suggestion
      return
```